collections/the-cents-club.csv

```diff
@@ -0,0 +1,100 @@
+1cent
```

Comment:
bad file.
example
1cent,5243523452345234525234235435

read readme.md file

Reply:
Hi,

Sorry, I could not understand what the issue was.
Can you please help me here so I can fix the issue and resubmit.

Thanks!!

Reply:
I followed these instructions:
<img width="653" alt="image" src="https://user-images.githubusercontent.com/77735317/176115909-5b669091-706a-4b0e-bb2f-5dfb427bfa05.png">